zulipterminal/cli/run.py

```diff
@@ -392,6 +478,40 @@ def list_themes() -> str:
     )
 
 
+def list_realms() -> str:
+    valid_orgs = [
+        file.parent.name for file in Path(ZULIP_CONFIG_PATH).glob("*/zuliprc")
+    ]
+    if len(valid_orgs) == 0:
+        exit_with_error("No organizations found in zuliprc config folder.")
+    text = "The following organizations are available:\n  "
+    return text + "\n  ".join(valid_orgs)
+
+
+def path_to_realm_zuliprc(realm_name_prefix: str) -> str:
+    if len(realm_name_prefix) < 3:
+        exit_with_error("Organization name prefix must be at least 3 characters long")
+    matching_dirs = [
+        dir
+        for dir in Path(ZULIP_CONFIG_PATH).glob("*/")
+        if dir.name.startswith(realm_name_prefix)
+    ]
+    if len(matching_dirs) > 1:
+        matching_dirs_string = "\n".join(str(dir) for dir in matching_dirs)
+        exit_with_error(
+            f"Found multiple organizations starting with '{realm_name_prefix}':\n"
+            f"{matching_dirs_string}\n"
+        )
```

Comment:
I'm not sure how feasible this is, but should we consider prompting the user to select which directory to log into from the matched directories?